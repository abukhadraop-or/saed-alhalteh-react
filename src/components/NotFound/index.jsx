import React from 'react';

/**
 * Page not found message.
 *
 * @return {JSX.Element} NotFound component.
 */
function NotFound() {
  return <h1>Page Not Found</h1>;
}

export default NotFound;
