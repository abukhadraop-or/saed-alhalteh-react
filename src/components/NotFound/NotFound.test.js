import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import NotFound from '.';

Enzyme.configure({ adapter: new Adapter() });

describe('<NotFound /> tests', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<NotFound />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
