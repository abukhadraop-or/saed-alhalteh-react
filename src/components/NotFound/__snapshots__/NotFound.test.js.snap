// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> tests matches the snapshot 1`] = `
<h1>
  Page Not Found
</h1>
`;
